ext_emconf.php

```diff
@@ -11,9 +11,9 @@
         'depends' => [
             'news' => '',
             'yoast_seo' => '',
-            'typo3' => '8.7.0-8.7.99'
+            'typo3' => '7.6.31-8.7.99'
```

Comment:
Is there a reason the TYPO3 minimum version is higher than in https://github.com/georgringer/news/blob/6.3.0/composer.json#L21?